### DISCUSSION TITLE
Access the log database from Power BI

### DISCUSSION THREAD

**Discussion by narrsam** [datalust/seq-tickets#1839] [Help]
Hi!

Can the log database in Seq be accessed by Power BI, or any other data visualization platform?

**Comment by liammclennan** [comment]
Hi @narrsam ,

Certainly. The PowerBI Web connector can be used to fetch data from Seq. See https://learn.microsoft.com/en-us/power-query/connectors/web/web#load-web-data-using-an-advanced-url.

**Reply by narrsam** [reply]
Thank you for the swift reply!
Can you give an example on what the URL parts and the HTML header parameters should look like ?

**Reply by liammclennan** [reply]
![image](https://github.com/datalust/seq-tickets/assets/271514/99c42442-9c75-4f81-86d6-668271424781)

The url should be something like 

`http://localhost:5341/api/data?q=select%20*%20from%20stream%20limit%2010&rangeStartUtc=2023-05-11T00%3A00%3A00.000Z&timeoutMS=600000&format=text%2Fcsv`

or 

`http://localhost:5341/api/events?count=30&shortCircuitAfter=100`

See the [API docs](https://docs.datalust.co/docs/server-http-api) to help you design a request that matches what you want (or you can look in the network tab of developer tools while you are using Seq). 

You also need to provide an API key with read permission or better.
